pkg/monitor/dynkeepalived.go

```diff
@@ -30,7 +30,7 @@ const (
 )
 
 var (
-	gBootstrapIP string
+	gBootstrapIP string = ""
```

Comment:
Is this necessary? Strings already default to "".

Reply:
Done